packages/host/app/components/operator-mode/code-submode/spec-preview.gts

```diff
@@ -222,6 +223,19 @@ class SpecPreviewContent extends GlimmerComponent<ContentSignature> {
     return !this.args.canWrite && this.args.cards.length === 0;
   }
 
+  @action viewSpecInstance() {
+    if (!this.args.selectedId) {
+      return;
+    }
+
+    try {
+      const selectedUrl = new URL(this.args.selectedId);
+      this.operatorModeStateService.updateCodePath(selectedUrl);
+    } catch (error) {
+      console.error('Failed to open spec instance:', error);
```

Comment:
Is this an expected circumstance? If not, I don't think we need the try/catch. We can just let the failure bubble. If so, we should have test coverage for it.